pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation_test.go

```diff
@@ -341,16 +342,27 @@ func TestExtractLibInfo(t *testing.T) {
 			},
 		},
 		{
-			name:              "python",
-			pod:               common.FakePodWithAnnotation("admission.datadoghq.com/python-lib.version", "v1"),
-			containerRegistry: "registry",
+			name:                "python",
```

Comment:
It turns out our tests rely on `mutate_unlabelled` being `on` by default, where this is not the case in production/reality. 

I added duplicate test cases with the flag disabled for this one to make sure we were going to filter out the pod at the top level `inject` function while _this test_ only validates what libraries we would inject regardless of that check. :shrug: